src/rsz/src/RecoverPower.cc

```diff
@@ -380,12 +380,10 @@ RecoverPower::downsizeCell(LibertyPort *in_port,
            float drive2 = port2->driveResistance();
            ArcDelay intrinsic1 = port1->intrinsicDelay(this);
            ArcDelay intrinsic2 = port2->intrinsicDelay(this);
-           return drive1 < drive2
-             || ((drive1 == drive2
-                  && intrinsic1 > intrinsic2)
-                 || (intrinsic1 == intrinsic2
-                 || (intrinsic1 == intrinsic2
-                     && port1->capacitance() > port2->capacitance())));
+
+           return ( drive1 < drive2 || 
+		    (drive1 == drive2 && intrinsic1 > intrinsic2) ||
+		    (intrinsic1 == intrinsic2 && port1->capacitance() > port2->capacitance()));
```

Comment:
Is this right?  If drive1 > drive2 and  intrinsic1 == intrinsic2 then you will use port cap not drive strength.

I tend to use std::tie in these cases (see operator< example in https://en.cppreference.com/w/cpp/utility/tuple/tie)

Reply:
I tested with 10+ designs and we don't seem to need the capacitance business. ORFS is also running here with this code change + call to recover_power added to cts.tcl 

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1233